tests/test_adversary.py

```diff
@@ -69,7 +69,6 @@ def test_adversary_with_model(input_data, target_data, perturbation):
 
     output_data = adversary(input_data, target_data, model=model)
 
-    parameters.assert_called_once()
```

Comment:
Thou shalt not change tests. Create a new test?

Reply:
The test case is tailored for the previous API and we have to change it, otherwise it fails.